### ISSUE TITLE
Did you try to run stable-diffusion utilizing the container using your legacy card?

### ISSUE THREAD

**Issue by Rolli700** [crazylinuxer/tensorflow-rocm-legacy#1]
I'm struggeling for days now trying to install rocm-dkms for my RX580 ... changing the kernel, trying different versions of anything and everything, always failing.
Wondering if it's possible to utilize your container therefor.

**Comment by kvirikroma** [comment]
@Rolli700 yes, I think it is possible, but probably you should change the Dockerfile depending on your purposes. I've done this repo for myself some time ago and successfully used it to train some neural networks, but some things are hard-coded here (like python packages to install). So you should consider this repo not as a pack of scripts that work out-of-the-box, but more like a template collection. And one more important thing I should mention: the only reason why this works for old AMD GPUs is because it builds from really old ROCm version (check the 1st line of the Dockerfile)

And sorry for not answering for so long, I should probably check my mail more often haha
